src/Avalonia.Controls/Generators/TreeContainerIndex.cs

```diff
@@ -34,7 +34,12 @@ public class TreeContainerIndex
         /// <summary>
         /// Gets the currently materialized containers.
         /// </summary>
-        public IEnumerable<IControl> Items => _containerToItem.Keys;
+        public IEnumerable<IControl> Containers => _containerToItem.Keys;
```

Comment:
This is a breaking change, add changes to the Wiki when this is merged.

Reply:
[This? ](https://github.com/AvaloniaUI/Avalonia/wiki/Breaking-Changes#treecontainerindex-in-treeview)